src/test/bash/run-prod-e2e.sh

```diff
@@ -10,23 +10,23 @@
 
 set -ev
 
-sed -i "s|new plugin.BaseHrefWebpackPlugin({ baseHref: '/' })|new plugin.BaseHrefWebpackPlugin({ baseHref: '/managementportal/' })|" webpack/webpack.dev.js
-sed -i "s|baseUrl: 'http://localhost:8080/',|baseUrl: 'http://localhost:8080/managementportal/',|" src/test/javascript/protractor.conf.js
-sed -i "s|contexts: prod|contexts: dev|" src/main/resources/config/application-prod.yml # set liquibase context to dev so it loads demo data
-./gradlew bootRepackage -Pprod buildDocker -x test
-docker-compose -f src/main/docker/app.yml up -d # spin up production mode application
-# wait for app to start up
-echo "Waiting for application startup"
-until curl -s http://localhost:8080/managementportal/ > /dev/null
-do
-  echo -n "." # waiting
-  sleep 2
-done
-echo ""
-echo "Application is up"
-docker-compose -f src/main/docker/app.yml logs # show output of app startup
-yarn e2e # run e2e tests against production mode
-docker-compose -f src/main/docker/app.yml down -v # clean up containers and volumes
-git checkout src/test/javascript/protractor.conf.js
-git checkout webpack/webpack.prod.js
-git checkout src/main/resources/config/application-prod.yml
+# only run on the release branch's push and pull_request events
+if [[ $TRAVIS_BRANCH == release-* ]] || [[ $TRAVIS_PULL_REQUEST_BRANCH == release-* ]]
```

Comment:
Perhaps run on the `master` branch as well, to see if there are any strange merge issues?

Reply:
The `TRAVIS_PULL_REQUEST_BRANCH` environment variable is set for builds on the merge result, with its value set to the name of the source branch, so by the time the code gets in to master it should have already been tested. I'm not opposed to running on master again though if you feel it would bring added value. What do you think?

Reply:
On the other hand, using `if [[ $TRAVIS_BRANCH == release-* ]] || [[ $TRAVIS_BRANCH == master ]]` will trigger on the PR build regardless of the source branch. Our current flow only merges into master from release branches, but if that would change in the future, then using this expression would mean we don't need to change this script. I'll update it!